bin/build-locales

```diff
@@ -50,15 +50,15 @@ poFiles.forEach((pofile) => {
     var defineLocale = null;
     try {
       const momentLocale = locale.replace('_', '-').toLowerCase();
+      // Check for the locale first; if it doesn't exist we don't have
+      // a moment locale that matches.
+      fs.accessSync(`./node_modules/moment/locale/${momentLocale}.js`);
```

Comment:
To make the access check more effective, you could share the same variable by doing:
````js
const localeModule = `moment/locale/${momentLocale}.js`;
fs.accessSync(`./node_modules/${localeModule}`);
// ...
defineLocale = new Function(`require('${localeModule}');`);
````